web/src/components/metrics/single-metrics-total-content.tsx

```diff
@@ -56,22 +55,7 @@ export const SingleMetricsTotalContent: React.FC<SingleMetricsTotalContentProps>
 }) => {
   const { t } = useTranslation('plugin__netobserv-plugin');
 
-  let filtered = [...topKMetrics];
-  if (showOthers) {
-    const others = {
-      name: othersName || t('Others'),
-      values: _.cloneDeep(totalMetric.values),
-      aggregateBy: 'dnsRCode',
-      stats: totalMetric.stats
-    } as GenericMetric;
-    filtered.forEach(m => {
-      for (let i = 0; i < m.values.length; i++) {
-        others.values[i][1] -= m.values[i][1];
-      }
-    });
-    filtered.push(others);
-  }
-  filtered = filtered.slice(0, limit);
+  const filtered = topKMetrics.filter(m => showOthers || (othersName && m.name !== othersName)).slice(0, limit);
```

Comment:
this change too pls explain is it relevant to DNSErrno change ?

Reply:
same as https://github.com/netobserv/network-observability-console-plugin/pull/425#discussion_r1379771282